### ISSUE TITLE
[Feature Request] Support Tachiyomi Extensions repo

### ISSUE THREAD

**Issue by saud-97** [hkalexling/Mango#304]
**Is your feature request related to a problem? Please describe.**
To increase the range of possible plugins that can be used with Mango. Tachiyomi has a a very big user base with 1k+ extensions.

**Describe the solution you'd like**
Add Tachiyomi Extensions compatibility.

**Describe a small use-case for this feature request**
The feature will serve as extra sources of manga.

**Additional context**
[Tachiyomi Extensions](https://github.com/tachiyomiorg/tachiyomi-extensions)

**Comment by hkalexling** [comment]
Hey thanks for the FR, but I don't see how this can be done. To interact with the extensions we will need to either ship a Java VM in our binary or require our users to have Java available on their systems.

**Comment by saud-97** [comment]
Yes even if it involved some preparation from user side, like maybe enable adb to access the extensions or some other alternative?

Those who are interested in Mango will do their best to make it work best like me. I wouldn't mind if I needed to do some setup to access them.

**Comment by z8512** [comment]
@Saud-97 Just use tachidesk. Problem fixed

**Comment by Tzrlk** [comment]
> @Saud-97 Just use tachidesk. Problem fixed

Neat, they even have a [server-specific implementation](https://github.com/Suwayomi/Tachidesk-Server) that a mango plugin could theoretically be a client for.
